cmd/query/app/handler.go

```diff
@@ -128,6 +132,21 @@ func (aH *APIHandler) RegisterRoutes(router *mux.Router) {
 	// TOOD - remove this when UI catches up
 	aH.handleFunc(router, aH.getOperationsLegacy, "/services/{%s}/operations", serviceParam).Methods(http.MethodGet)
 	aH.handleFunc(router, aH.dependencies, "/dependencies").Methods(http.MethodGet)
+
+	router.PathPrefix("/static").Handler(http.FileServer(http.Dir(staticAssetsRoot)))
```

Comment:
I think it's better to move this to another handler, separate from the API handler.